src/Nethermind/Nethermind.Synchronization/ParallelSync/MultiSyncModeSelector.cs

```diff
@@ -29,7 +29,7 @@ public class MultiSyncModeSelector : ISyncModeSelector, IDisposable
         /// <summary>
         /// Number of blocks before the best peer's head when we switch from fast sync to full sync
         /// </summary>
-        public const int FastSyncLag = 32;
+        public const int FastSyncLag = 2;
```

Comment:
Is this safe? Maybe this should be 2 on only when beam sync is on?

Reply:
yeah it is safe based on my analysis